examples/mollie-credit-card/api-types/storeApiTypes.d.ts

```diff
@@ -7317,9 +7317,13 @@ export type operations = {
       "sw-language-id"?: string;
     };
     body: {
+      /**
+       * Id - is used to add products to the cart.
+       * referencedId - is used to add promotions to the cart or create new line of the same item
+       */
       // TODO: [OpenAPI][addLineItem] - add proper request body type with required fields
       items: Array<{
-        id: string; // TODO: [OpenAPI][addLineItem] - check if this is used at all?
+        id?: string; // TODO: [OpenAPI][addLineItem] - check if this is used at all?
```

Comment:
You need to provide a schema patch to fix it; this will be removed in the next generation. This file should not be modified manually.

I'll provide a commit here to show the way of adding a patch.